packages/dvilela/services/memeooorr/service.yaml

```diff
@@ -165,14 +165,14 @@ config:
   twikit_email: ${TWIKIT_EMAIL:str:null}
   twikit_password: ${TWIKIT_PASSWORD:str:null}
   twikit_cookies: ${TWIKIT_COOKIES:str:null}
-  twikit_cookies_path: ${TWIKIT_COOKIES_PATH:str:/tmp/twikit_cookies.json}
   twikit_disable_tweets: ${TWIKIT_DISABLE_TWEETS:bool:false}
   twikit_skip_connection: ${TWIKIT_SKIP_CONNECTION:bool:false}
+  store_path: ${STORE_PATH:str:null}
 ---
 public_id: dvilela/kv_store:0.1.0
 type: connection
 config:
-  db_path: ${DB_PATH:str:/logs/memeooorr.db}
+  store_path: ${STORE_PATH:str:null}
```

Comment:
@jmoreira-valory you can see 2 instances of STORE_PATH...let's see if this works as is